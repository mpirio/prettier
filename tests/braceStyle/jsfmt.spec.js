run_spec(__dirname, ["babel", "flow", "typescript"]);
run_spec(__dirname, ["babel", "flow", "typescript"], { braceStyle: "stroustrup" });
