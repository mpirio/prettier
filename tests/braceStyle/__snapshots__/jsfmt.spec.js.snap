// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests.js 1`] = `
====================================options=====================================
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
if (a) { b = 2; } else { b = 3; }

if (a) {
    b = 2;
} else {
    b = 3;
}

if (a) {
    b = 2;
}
else {
    b = 3;
}

do {
    b++;
} while (a)

try {
    a = 1;
} catch (error) {
    console.error(error);
} finally {
    b = 2;
}

try {
    a = 1;
}
catch(error) {
    console.error(error);
}
finally {
    b = 2;
}
=====================================output=====================================
if (a) {
  b = 2;
} else {
  b = 3;
}

if (a) {
  b = 2;
} else {
  b = 3;
}

if (a) {
  b = 2;
} else {
  b = 3;
}

do {
  b++;
} while (a);

try {
  a = 1;
} catch (error) {
  console.error(error);
} finally {
  b = 2;
}

try {
  a = 1;
} catch (error) {
  console.error(error);
} finally {
  b = 2;
}

================================================================================
`;

exports[`tests.js 2`] = `
====================================options=====================================
braceStyle: "stroustrup"
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
if (a) { b = 2; } else { b = 3; }

if (a) {
    b = 2;
} else {
    b = 3;
}

if (a) {
    b = 2;
}
else {
    b = 3;
}

do {
    b++;
} while (a)

try {
    a = 1;
} catch (error) {
    console.error(error);
} finally {
    b = 2;
}

try {
    a = 1;
}
catch(error) {
    console.error(error);
}
finally {
    b = 2;
}
=====================================output=====================================
if (a) {
  b = 2;
}
else {
  b = 3;
}

if (a) {
  b = 2;
}
else {
  b = 3;
}

if (a) {
  b = 2;
}
else {
  b = 3;
}

do {
  b++;
} while (a);

try {
  a = 1;
}
catch (error) {
  console.error(error);
}
finally {
  b = 2;
}

try {
  a = 1;
}
catch (error) {
  console.error(error);
}
finally {
  b = 2;
}

================================================================================
`;
